### PR TITLE
fix: add mimetype for eml and msg

### PR DESCRIPTION
**Pull request by kesselb**
For https://github.com/nextcloud/files_emailviewer/issues/1

## Summary

Add mimetype mapping for eml and msg files

## TODO

- [x] CI
- [x] Backport 28

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]

**Review by st3iny — APPROVED** [review]
:elephant:

**Review by ChristophWurst — APPROVED** [review]
Tested and works, too

Uploaded MSG files are now detected with ``<d:getcontenttype>application/x-ole-storage</d:getcontenttype>``

File: lib/private/Repair/RepairMimeTypes.php [hunk 1]
```diff
@@ -237,6 +237,15 @@ private function introduceEnhancedMetafileFormatType() {
 		return $this->updateMimetypes($updatedMimetypes);
 	}
 
+	private function introduceEmlAndMsgFormatType() {
+		$updatedMimetypes = [
+			'eml' => 'message/rfc822',
+			'msg' => 'application/x-ole-storage',
```
Comment (ChristophWurst) [inline lib/private/Repair/RepairMimeTypes.php:243]: yet I'm wondering if it should be `application/vnd.ms-outlook`

**Review by AndyScherzinger — APPROVED** [review]

**Pull request merged**